packages/desktop-client/src/components/reports/DateRange.tsx

```diff
@@ -15,7 +17,7 @@ type DateRangeProps = {
 
 function checkDate(date: string) {
   const dateParsed = new Date(date);
-  if (dateParsed.toString() !== 'Invalid Date') {
+  if (dateParsed.toString() !== t('Invalid Date')) {
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this one should not be translated.